java/fury-core/src/test/java/io/fury/serializer/collection/CollectionSerializersTest.java

```diff
@@ -477,7 +476,7 @@ public void testJavaSerialization() {
 
     assertSame(
         fury.getClassResolver().getSerializer(setClass).getClass(),
-        GuavaCollectionSerializers.ImmutableSortedSetSerializer.class);
+        UnmodifiableSerializers.UnmodifiableCollectionSerializer.class);
```

Comment:
Hi @nandakumar131, guava usage in tests is OK. actually we need the input object to be a guava list to test GuavaSerializers. In the long run, we may mark guava as an maven optional dependency and only mark itas an compile dependency for test

Reply:
Sure, will revert this change!